pkg/cmd/cli/cmd/process.go

```diff
@@ -80,6 +80,15 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 		return kcmdutil.UsageError(cmd, "Must pass a filename or name of stored template")
 	}
 
+	if kcmdutil.GetFlagBool(cmd, "parameters") {
+		mutuallyExclusive := []string{"value", "labels", "output", "output-version", "raw", "template"}
+		for _, flag := range mutuallyExclusive {
```

Comment:
Inline the constant into the range statement ( `range []string{...}` ) for clarity